site/en/docs/webstore/manage/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manage your Chrome Web Store Item"
+date: 2021-08-17
+description: >
+  How to manage your extension in the Chrome Web Store.
+---
+
+This page describes how to manage an extension that was previously submitted on the Chrome Web Store.
+
+## About the lifecycle of an item in the Chrome Web Store 
+
+All extensions go through an automated review process. In some instances a manual review is required, specially when sensitive permissions are requested. For this reason review times and/or approval times can take longer. A Chrome Web Store item goes through several stages, as depicted in the following diagram:
```

Comment:
This is totally bikeshedded, but could you put each new sentence on its own line? I don't want wrapping, but whole paragraphs makes restructuring still a bit tricky.